resources/js/processes/modeler/components/inspector/AssignExpression.vue

```diff
@@ -60,7 +60,7 @@
           </div>
           <div class="col-9 p-0" style="cursor:grab" >
             <div class="displayed-expression">
-              {{ assignment.displayExpression }}<span>...</span>
+              {{ assignment.displayExpression }}<span v-if="assignment.expression.length > 14">...</span>
```

Comment:
Could you use `text-truncate` bootstrap class instead of `v-if`for the ellipsis `...`

Reply:
Didn't even know that existed! I was able to remove additional code in replace of that class! Thank you!